### PR TITLE
Documentation: add link to release announcements

### PR DESCRIPTION
**Pull request by jrfnl**
I was having trouble finding the older ones, so figured having a link to the page somewhere might be useful.

### CODE REVIEW

File: public/documentation.html [hunk 1]
```diff
@@ -43,6 +43,7 @@ <h1>Documentation for PHPUnit</h1>
       <li><a href="https://phpunit.readthedocs.io/es/latest/">Spanish</a></li>
      </ul>
      <p>Documentation for PHPUnit 6.5 (and older) is archived at <code>https://phpunit.de/manual/6.5/en/index.html</code>. Simply replace <code>6.5</code> with the version number you are looking for. Simply replace <code>en</code> with <code>fr</code>, <code>pt_br</code>, <code>ja</code>, or <code>zh_cn</code> to access the French, Brazilian Portuguese, Japanese, or Simplified Chinese translation, respectively.</p>
+     <p>Important information about changes in major releases can be found in the <a href="announcements/">release announcements</a>, for changes in minor versions more information can be found in the <a href="https://github.com/sebastianbergmann/phpunit/wiki/">wiki</a>.</p>
```
Comment (sebastianbergmann) [inline public/documentation.html]: "for changes in minor versions more information can be found in the wiki" is not correct. This information can be found in the `ChangeLog-X.Y.md` files.

Comment (jrfnl) [reply public/documentation.html]: @sebastianbergmann I'd be happy to change this, however IMO those are very unwieldy to work with as older changelogs are deleted from the master branch which means in effect that the user would need to switch branches a lot of times to be able to find all those. As I did find them in the wiki as well (after a google search), that seemed the more user friendly option to point people to.

Comment (sebastianbergmann) [reply public/documentation.html]: The wiki is no longer used for release announcements or ChangeLogs. If you want to link to changes in minor releases then you have to use `https://github.com/sebastianbergmann/phpunit/blob/8.1/ChangeLog-8.1.md#810---2019-04-05` for PHPUnit 8.1, for example.

**Comment by sebastianbergmann** [comment]
Thank you for your contribution. I appreciate the time you invested in preparing this pull request. However, I have decided not to merge it.

I will take care of this shortly in #45.

**Comment by jrfnl** [comment]
Seen the commit with the additional changelog links. Awesome! :tada: Much appreciated.

**Pull request closed without merging**